Lab1.ipynb

```diff
@@ -289,66 +358,197 @@
+    }
+   ],
+   "source": [
+    "x = np.linspace(0,80,81)\n",
+    "rate = [1,5,10,20,50]\n",
+    "\n",
+    "plt.rcParams[\"figure.figsize\"] = (15,10)\n",
+    "fig, ax = plt.subplots(1,1)\n",
+    "plt.tick_params(labelsize = 10)\n",
+    "plt.xticks(np.linspace(0,80,17))\n",
+    "plt.xlim(0,80)\n",
+    "plt.ylim(0,0.4)\n",
+    "plt.title('Poisson Distribution Step Plot for given Rate Parameters', fontsize = 'large')\n",
+    "plt.xlabel('Value',fontsize='large')\n",
+    "plt.ylabel('Probability',fontsize='large')\n",
+    "\n",
+    "for val in rate:\n",
+    "    pmf = stats.poisson.pmf(x,val)\n",
+    "    plt.step(x,pmf,linewidth=5,alpha = 0.7,label = f'{val}')\n",
+    "plt.legend(title='Rate Parameter',title_fontsize = 'large',fontsize='xx-large')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f0b665b7-c3e1-429d-b1d0-fde756f348d8",
+   "metadata": {},
+   "source": [
+    "#### C.\n",
+    "\n",
+    "Suppose my signal-less distribution produced a rate parameter of 10. Given this, what is the probability of obtaining a measurement of 45 or greater from the signal-less distribution?"
+   ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "id": "29b1ac1b-94f5-4264-b720-11ac437c06ff",
+   "execution_count": 157,
+   "id": "360f8e8e-80d0-4d8c-88f1-373138fc6a31",
    "metadata": {},
-   "outputs": [],
-   "source": []
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "1.0464947441174539e-16"
+      ]
+     },
+     "execution_count": 157,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "s=10\n",
+    "my_measurement = 45\n",
+    "\n",
+    "stats.poisson.sf(my_measurement,s)"
+   ]
   },
   {
    "cell_type": "markdown",
-   "id": "1e38093a-8fc8-4166-b845-d42dfc4a1594",
+   "id": "41079aad-e73d-4b18-b460-a7ef88f266cb",
    "metadata": {},
    "source": [
-    "## Non-continuous distributions (Poisson)"
+    "Now I see that the measurement of 45 would be extremely unlikely from a signal-less distribution, and has a sigma value of 8.2 associated with it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 158,
+   "id": "6e13b138-0f70-4c3a-b47c-ce82882e575c",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "8.209536151601387"
+      ]
+     },
+     "execution_count": 158,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "stats.norm.ppf(stats.poisson.cdf(my_measurement,s))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d13542e5-17fc-4534-8669-4175ad4d08c0",
+   "metadata": {},
+   "source": [
+    "#### D.\n",
+    "\n",
+    "Since probabilities and sigmas are discrete, this means that there will be a certain value measurement that crosses the 5-Sigma barrier, while 1 less than that value would not. For experiments this would create a sort of 'benchmark' value that you are hoping to produce. \n",
```

Comment:
Here you already run with the claim that probabilities and sigmas are discrete, but I think for the sake of this lab you'd also want to explain why that is.